docs/to_clarify.md

```diff
@@ -1 +1,23 @@
-1. Why does identification verification require "booking document + passport + credit card authorization"?
+1. Stated under the section "Problem Statement" that the solution must be "safe and secured to protect guests' privacy". What exactly does this requirement mean, for example, are there certain security standards that the solution must adhere to?
+
+1. Stated under the section "Possible Use Cases" that the solution should "enhance back-end operational capabilities and business intelligence" which will be used to "create accurate guest profiles and provide personalized and targeted guest experiences". In what manner is the solution expected to do this?
+
+1. Stated under the section "Possible Use Cases" that the solution should "allow data transference capabilities, from room requests collected on the booking engine, to the check-in process till checkout".
+
+    Does this mean that the "digital mobile check-in and check-out solution" is expected to do more than digitising the checking-in and checking-out processes; it has to integrate with other platforms to provide other additional services, beginning from the booking of hotel rooms till checkout?
+
+1. Stated under the section "Possible Use Cases" that the solution should "begin their check-in journey before arrival and complete it upon arrival".
+
+    a. What needs to be done during the checking-in process?
+
+    b. Which part of the process can guests complete up to before arrival, and what else do they need to do to complete the check-in process upon arrival?
+
+    c. On the same note, what needs to be done during the checking-out process?
```

Comment:
Can this question be asked **after** they run through with us (if any) the current check-in/out process? I feel that these questions might be answered after that discussion.

Reply:
Yup, sounds good. What do you think if I split our content into 2 segments:

```
## Questions To Ask

### In Forum

// most of our questions

### Additional Questions If Unanswered Above

// this segment
```

Reply:
> Yup, sounds good. What do you think if I split our content into 2 segments:
> 
> ```
> ## Questions To Ask
> 
> ### In Forum
> 
> // most of our questions
> 
> ### Additional Questions If Unanswered Above
> 
> // this segment
> ```

I think this is good. Thanks!

Reply:
Sounds good.